src/modules/DASDController.rb

```diff
@@ -465,84 +466,55 @@ def ReportActivationError(channel, ret)
       when 0
 
       when 1
-        Report.Error(
-          Builtins.sformat(
-            # error report, %1 is device identification
-            _("%1: sysfs not mounted."),
-            channel
-          )
-        )
+        # error report, %s is device identification
+        Report.Error(format(_("%s: sysfs not mounted."), channel))
```

Comment:
Ok, I did that change because Rubocop was complaining about the class lenght. So, instead to increase the limit I decide to save LOC's putting the `Report.Error` in oneline. And... you know, I though: "I should took the opportunity to replace this `Builtins.sformat`..." 

But if your think it is better to keep "%1", I'll bring back the p`Builtins.sformat` :)

Reply:
well, I prefer to use format in new code. But modify all that translatable string just for that is too much work for translators for nothing. So if you modify string or create new one, format is what you should use, but changing just format characters is for me unnecessary work for translators.